### DISCUSSION TITLE
umi4页面能正常运行，但是代码就是标红是为啥？

### DISCUSSION THREAD

**Discussion by liushunli** [umijs/umi#11705] [Q&A]
![image](https://github.com/umijs/umi/assets/12014406/daa179eb-0c87-4263-8bf6-14534d607be2)

umi4页面能正常运行，但是代码就是标红是为啥？

**Comment by fz6m** [comment]
确保你的 vscode 版本是最新的，并且正确安装了依赖，当前工作区拥有正确的 `tsconfig.json` ，如有红线提示问题，可以 hover 上去查看具体原因进一步自行排查。
